app/lib/ability.rb

```diff
@@ -20,6 +20,9 @@ def initialize(user)
         cannot :manage, StateFileDependent
         cannot :manage, StateId
         cannot :manage, EfileSubmission, id: EfileSubmission.for_state_filing.pluck(:id)
+        cannot :manage, EfileError do |error|
```

Comment:
restrictions on who can see and make edits to these errors in the abilities file since it can impact what the client sees on the return-status page